### ISSUE TITLE
Low contrast on date/links

### ISSUE THREAD

**Issue by LiamBull** [LiamBull/blog#1]
There is insufficient contrast on the date and links sections, resulting in poor accessibility.

**Comment by LiamBull** [comment]
https://github.com/LiamBull/blog/commit/6d4b519caf33eea1aa909c36d888d9c683eca1e5
